### ISSUE TITLE
Implement the Twitter Pynq Provider

### ISSUE THREAD

**Issue by heynemann** [heynemann/pynq#18]
Implement the Twitter Pynq Provider as a tutorial on writing your own Pynq Provider.

**Comment by glauco** [comment]
I think that the name of it provider would be 'Microblogging Provider' and would support Twitter and other sites that use Laconica. I think it because the APIs are very similar.

**Comment by heynemann** [comment]
True. At the point where we implement it we can make sure we use the proper API.
